documentation/UserGuide.md

```diff
@@ -260,3 +261,15 @@ TinyCurrentLogger value: customLogger during: [
 	TinyCurrentLogger value record: 'Test2'
 ]
 ```
+
+## Clear your logger
+
+Each logger can understand the method `#clearLogger`. This method will have as effet to clear the output of the loggers. The actual effect can be slightly different depending on the kind of logger:
```

Comment:
"can understand" -> understands

effet -> effect

"The actual effect can be slightly different" --> "The actual effect is different depending on"

Reply:
Also, maybe `clearLog` is a better message? The log is the holder of the messages, whereas the logger is the thing that records the messages?